### PR TITLE
Image link repair

### PR DESCRIPTION
**Pull request by HoshinoKyouna**

### CODE REVIEW

File: SSH/README.md [hunk 1]
```diff
@@ -8,59 +9,56 @@
...
+## SSH password login
 
 For first time login, users could use the following command to login to the system via SSH. The command is as follows:
 
 ```
 ssh -p [port] [account]@[ip]
 ```
 
-![img](https://github.com/cuhk-haosun/doc-manual/blob/image-for-ssh/SSH/image/ssh%20powershell.gif?raw=true)
+<img title="" src="https://github.com/HoshinoKyouna/doc-manual-pub/blob/image-link-repair/SSH/image/ssh%20powershell.gif?raw=true" alt="img" data-align="inline">
```
Comment (Milokita) [inline SSH/README.md]: You're using absolute link, pls using relative link:

```
<img title="" src="image/ssh%20powershell.gif?raw=true" alt="img" data-align="inline">
```

**Review by Milokita — CHANGES_REQUESTED** [review]
pls refer the inline comment

**Comment by HoshinoKyouna** [comment]
Sorry for lacking the knowledge about it. I will take it.

**Comment by Milokita** [comment]
pls don't be too hard on yourself, I should give you a ref

**Review by HoshinoKyouna — COMMENTED** [review]
Absolute links were changed into relative ones.

**Review by Milokita — APPROVED** [review]

**Pull request merged**